cassdegrees/ui/views/student.py

```diff
@@ -100,6 +101,23 @@ def student_create(request):
         response = redirect(student_edit)
         response['Location'] += '?plan='
         return response
+    # If a plan name is given, copy that plan
+    elif plan_to_duplicate:
+        plan = decompress(request.session.get('plan:' + plan_to_duplicate, None))
+        print(plan)
```

Comment:
Make sure you aren't logging this when you merge

Reply:
Yeah I just noticed this, removed.